network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgTunnel.kt

```diff
@@ -82,7 +80,6 @@ class RealWgTunnel @Inject constructor(
                 }
         } catch (e: Throwable) {
             logcat(LogPriority.ERROR) { "Error getting WgTunnelData: ${e.asLog()}" }
-            crashLogger.logCrash(CrashLogger.Crash(shortName = "m_netp_ev_key_registration_error", t = e))
```

Comment:
Incidentally removed. I've checked the crash reports and we know what we wanted and now how to fix it. See https://github.com/duckduckgo/Android/pull/3736